Tests/Fixtures/Document.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Resource\Tests\Fixtures;
+
+use Doctrine\ODM\PHPCR\HierarchyInterface;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Document implements HierarchyInterface
```

Comment:
i will have a look if i still need that class, as we removed the name/parent assertions.